src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Collapse, IconButton, TableCell, TableRow } from '@material-ui/core/';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+import ApplicantSubTable from './components/ApplicantSubTable';
+import HallSubTable from './components/HallSubTable';
+import './applicationRow.css';
+
+const ApplicationRow = ({ applicationDetails, labelId }) => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <TableRow key={applicationDetails.ApplicationID} onClick={() => setOpen(!open)}>
+        <TableCell align="center" component="th" id={labelId} scope="row">
+          {applicationDetails.ApplicationID}
+        </TableCell>
+        <TableCell align="center">{applicationDetails.EditorUsername}</TableCell>
+        <TableCell align="center">{applicationDetails.Applicants.length}</TableCell>
```

Comment:
Should this have a word identifying it `Applicants: 5`

Reply:
I'm not sure what you mean.
If you mean put the text in the cell with the number, then the answer is no, because the column header on the table tells you what the numbers in that column represent

Reply:
Oh, right, I saw that later on and should have removed this comment. Good!